### DISCUSSION TITLE
New user registration

### DISCUSSION THREAD

**Discussion by Royster63** [soundshed/soundshed-app#37] [Q&A] [answered]
Hi. I have attempted to create a new user account on the Soundshed Page but keep getting this message "Registration failed. Check email address and password are set and you are not using an email that's already registered."

**Comment by TiredRob** [comment]
Can't help, but I'm getting the same thing...

**Reply by Royster63** [reply]
Hi. When I posted this, I was trying to use the web based version. I have downloaded the Windows PC version now and log in is not required.

**Reply by webprofusion-chrisc** [reply]
Note that login is only required to save tones to the soundshed.com tone community, which is nothing to do with the positive grid tone cloud and is otherwise not necessary for any other functionality. The idea was it would let you sync your presets across devices etc but that's not a thing yet.

Yeah I haven't looked at this yet, I didn't notice the discussion notification.

**Reply by Porgytirebiter** [reply]
Have the same issue on Mac OSX Mojave. Can't create account on the web app or stand alone app.

**Reply by webprofusion-chrisc** [reply]
Yeah I'm probably going to remove sign in options on the next release, it doesn't do much anyway.

**Reply by Porgytirebiter** [reply]
Aside from that one little glitch it works very well. Nice work.

**Reply by genmce** [reply]
Same on windows 10 pc here.

**Reply by RocklandSally** [reply]
just tried on Win10 and no luck.. THEN I saw this thread..lol.. cheers for this.. cant wait for my Spark.. its in the mail

**Reply by morrissey007** [reply]
I'm having a similar problem, I'm using the app on Monterey and when I try to register or login it just sits there doing nothing, no errors, no login no messages.

**Reply by webprofusion-chrisc** [reply]
The sign in option has been removed from the app, some problems with the build are preventing me releasing the update just now. Basically the sign in feature does nothing so don't worry about it,

**Reply by justaxeme** [reply]
8 months later and we still can't register to share tones?

**Reply by justaxeme** [reply]
i Just grabbed the latest version and it still has the sign in option (that doesn't function).

**Reply by webprofusion-chrisc** [reply]
Yes.

**Reply by webprofusion-chrisc** [reply]
The macOS version can't be made to work on new macs so I think this installed version is basically dead. The web version is current however - it requires Google Chrome (or a browser that supports web bluetooth): https://app.soundshed.com/

**Comment by webprofusion-chrisc** [comment, marked as answer]
The sign in feature has now been removed from the latest version as it was unused.
